src/common.proto

```diff
@@ -5,8 +5,6 @@ package types;
 message FieldTypes {
   // Field type name
   string name = 1;
-  // Field type language code
-  string languageCode = 2;
   // The minScore will filter results which has lower certainty than the provided value.
   string minScore = 3;
```

Comment:
is it ok that now minScore is 3 and not 2

Reply:
also, consider reverting this change, as it is not backward compatible.
until we add versioningto the gen proto we should always backward support (and mention it should be removed)

Reply:
@eladiw good point about the backward compatibility.
So perhaps to leave the language code intact, for now, and just add it to template. 
Later we should remove it from FieldTypes

Reply:
Needs to be marked somehow as deprecated so it wont be used from now on.
check if there is an elegant way to mark it as such. (fallback plan will be a comment)

Reply:
please remove the [deprecated=true] tag and remove languageCode - we are not in production and more API changes are needed, so no need to be backward compatible at this point.

Reply:
We will push this and remove backward compatibility in the a following PR

Reply:
There is a good guide here on versioning and GRPC written [by the team behind the Envoy proxy](https://blog.envoyproxy.io/evolving-a-protocol-buffer-canonical-api-e1b2c2ca0dec). One thing to note is the use of the `reserved` keyword to park unused/deleted field numbers to avoid reuse by accident causing failures in the future -> https://developers.google.com/protocol-buffers/docs/proto3#reserved

Reply:
Update- this is fixed now